File: DifColl.Server/DTOs/MovieDto.cs
namespace DifColl.Server.DTOs
{
    public class MovieDto
    {
        public string Id { get; set; }

        public string Title { get; set; }

        public string Directors { get; set; }

        public string Genres { get; set; }

        public string ReleaseDate { get; set; }

        public string PosterPath { get; set; }

        public string Overview { get; set; }

        public float Rating { get; set; } // New
    }
}

File: difcoll.client/src/components/MovieDto.js
export default class MovieDto {
    constructor({
        id = '',
        title = 'Unknown Title',
        directors = 'Unknown',
        genres = 'Unknown',
        releaseDate = 'Unknown',
        posterPath = '',
        overview = 'No description available',
        rating = 0,
    }) {
        this.id = id;
        this.title = title;
        this.directors = directors;
        this.genres = genres;
        this.releaseDate = releaseDate;
        this.posterPath = posterPath;
        this.overview = overview;
        this.rating = rating;
    }
}
